static/app/components/events/groupingInfo/groupingVariant.tsx

```diff
@@ -105,7 +105,6 @@ function GroupingVariant({event, showGroupingConfig, variant}: GroupingVariantPr
     switch (variant.type) {
       case EventGroupVariantType.COMPONENT:
         component = variant.component;
-
```

Comment:
Total nit, feel free to ignore, but I actually think the space here is good, as it visually separates the `if`. (Also, bonus, if you leave it be, you'll be making the git blame on this file a micron simpler.)